css-brochure-form/styles.css

```diff
@@ -0,0 +1,62 @@
+form {
+  width: 60%;
+  margin-left: 20%;
+  margin-right: 20%;
+}
+
+h2 {
+  color: #ec5d57;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: bolder;
+  letter-spacing: 0.1rem;
+  text-align: center;
+}
+
+label {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-weight: bold;
+  color: #2b2b2b;
+  padding-left: 16px;
+  line-height: 1.75rem;
+}
+
+input {
+  font-family: AtlasGrotesk, sans-serif;
+  font-size: larger;
+  color: #999;
+  padding-left: 16px;
+  margin-bottom: 1.25rem;
+  border-color: #a6aaa9;
+  border-width: 1px;
+  border-radius: 0.5rem;
+  width: 100%;
+  height: 3.5rem;
+}
+
+select {
+  background-color: #f3f3f3;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-size: larger;
+  color: #333;
+  padding-left: 7px;
+  border-color: #a6aaa9;
+  border-width: 1px;
+  border-radius: 0.5rem;
+  width: 100%;
+  height: 3.5rem;
+}
+
+.button-right {
+  text-align: center;
+  background-color: #ec5d57;
+  border: none;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-weight: bold;
+  color: white;
+  height: 3rem;
+  width: 8rem;
+  border-radius: 30px;
+  margin-top: 1rem;
+  margin-left: 80%;
```

Comment:
❌ 

This margin is **_way_** too big. You should instead use the `text-align` property on the `div` parent of this button to align the button within that `div`.